modin/pandas/test/test_io.py

```diff
@@ -580,6 +586,84 @@ def test_read_csv_col_handling(
             **kwargs,
         )
 
+    # General Parsing Configuration
+    @pytest.mark.parametrize("dtype", [None, True])
+    @pytest.mark.parametrize("engine", [None, "python", "c"])
+    @pytest.mark.parametrize(
+        "converters",
+        [
+            None,
+            {
+                "col1": lambda x: np.int64(x) * 10,
+                "col2": pd.to_datetime,
+                "col4": lambda x: x.replace(":", ";"),
+            },
+        ],
+    )
+    @pytest.mark.parametrize("true_values", [["Yes"], ["Yes", "true"], None])
+    @pytest.mark.parametrize("false_values", [["No"], ["No", "false"], None])
+    @pytest.mark.parametrize("skiprows", [2, lambda x: x % 2])
+    @pytest.mark.parametrize("skipfooter", [0, 10])
+    @pytest.mark.parametrize("nrows", [123, None])
+    @pytest.mark.parametrize("names", [["c1", "c2", "c3", "c4"], None])
```

Comment:
By making so many parameters in one test, you make a very large matrix of test parameters, which greatly increases the testing time.

Reply:
+1, I think that we can separate this block of parameters in a separate test
```python
    @pytest.mark.parametrize("skiprows", [2, lambda x: x % 2])
    @pytest.mark.parametrize("skipfooter", [0, 10])
    @pytest.mark.parametrize("nrows", [123, None])
    @pytest.mark.parametrize("names", [["c1", "c2", "c3", "c4"], None])
```

Reply:
Tests were spitted into next groups:
1) `dtype`, 'engine` and `converters` and `skipfooter` (`read_csv` with `skipfooter` parameter behavior differs with different `engine`, these parameters should be tested together.
2) `true_values`, `false_values`, `skiprows`, `skipfooter`,  `nrows` and `names` (`true/false_values` should be tested together with `names` parameter because of https://github.com/modin-project/modin/issues/2446, `skipfooter` is also added here since this parameter is responsible for rows skipping as another parameters from this group)

Number of parsing parameters tests is reduced from 1728 to 168.